app/workers/functions.php

```diff
@@ -1,100 +1,315 @@
 <?php
 
+require_once __DIR__ . '/../worker.php';
+
+use Utopia\Queue;
+use Utopia\Queue\Message;
 use Appwrite\Event\Event;
-use Appwrite\Event\Func;
 use Appwrite\Messaging\Adapter\Realtime;
-use Appwrite\Resque\Worker;
 use Appwrite\Usage\Stats;
 use Appwrite\Utopia\Response\Model\Execution;
-use Cron\CronExpression;
 use Executor\Executor;
 use Utopia\App;
 use Utopia\CLI\Console;
 use Utopia\Config\Config;
 use Utopia\Database\Database;
-use Utopia\Database\DateTime;
 use Utopia\Database\Document;
 use Utopia\Database\ID;
 use Utopia\Database\Permission;
 use Utopia\Database\Query;
 use Utopia\Database\Role;
+use Utopia\Database\Validator\Authorization;
+use Utopia\Logger\Log;
+
+Authorization::disable();
+Authorization::setDefaultStatus(false);
+
+global $client;
```

Comment:
```suggestion
global $connection;
```

This should be a connection not a client.

Reply:
Renamed